src/dbus/common/CMakeLists.txt

```diff
@@ -38,6 +38,6 @@ target_link_libraries(otbr-dbus-common PUBLIC
     otbr-common
     openthread-ftd
     openthread-posix
-    ${DBUS_LINK_LIBRARIES}
+    ${DBUS_LIBRARIES}
```

Comment:
Should we add DBUS_LINK_LIBRARIES, in case these libraries are not in the standard path. I think this is common on macOS. The homebrew prefix could be under home.

Reply:
sorry for the typo, it should be `DBUS_LIBRARY_DIRS`

https://cmake.org/cmake/help/v3.10/module/FindPkgConfig.html